### DISCUSSION TITLE
I could not figure out how to use these buttons in the filter tool bar

### DISCUSSION THREAD

**Discussion by grotesque** [freeplane/freeplane#1444] [Help]
![image](https://github.com/freeplane/freeplane/assets/305091/7484e671-c3e5-4e3a-95a2-78404c021466)

1. "And Filter" button on filter tool bar
2. "Or Filter" button on filter tool bar
3. "Applies to filtered nodes" button on filter tool bar
Can somebody point to a tutorial or something that will help me understand. I was able to make sense of "Compose Filter" and its And and Or feature but not these.

___

Another issue is in following image
![image](https://github.com/freeplane/freeplane/assets/305091/fe92a7cc-c7ba-47a7-a89d-4061ee679c69)
The composed filter's name is cut out and isn't visible on hover. I think it should be visible as there is no way to know what the full condition is.
![image](https://github.com/freeplane/freeplane/assets/305091/9fe3348c-421a-40ea-8115-9f00177ccfd9)

**Comment by dpolivaev** [comment]
The buttons you ask about allow you to create a new filter condition as a boolean combination of the currently active filter condition and the condition currently selected in the toolbar condition editor.

**Comment by dpolivaev** [comment]
If you have more than one issue or question please submit separate issues. Otherwise, I can not manage it.

**Reply by dpolivaev** [reply]
I have created a separate issue https://github.com/freeplane/freeplane/issues/1445 for your second topic.
